http/api/transport.go

```diff
@@ -76,14 +77,32 @@ func decodeRequest(_ context.Context, r *http.Request) (interface{}, error) {
 	return msg, nil
 }
 
-// TODO: contact an auth provider
 func authorize(r *http.Request) (string, error) {
 	var apiKey string
+	var id string
+
 	if apiKey = r.Header.Get("Authorization"); apiKey == "" {
 		return "", errUnauthorizedAccess
 	}
 
-	return apiKey, nil
+	identityURL := os.Getenv("HTTP_ADAPTER_MANAGER_URL") + "/identity"
```

Comment:
This can be set at top-level, as a constant, instead of setting it per each request.